### PR TITLE
Setup Dynamic Form Templating

### PR DESCRIPTION
**Pull request by qwwqwwq**
- Adds initial form template for intent to file
- Adds static serving of the formly templates, and complimentary angular service.
- Changes /form path to include routeParam that determines which form is displayed & rendered.

### CODE REVIEW

File: winterfell/webapp/src/js/formRenderingService.js [hunk 1]
```diff
@@ -27,3 +27,29 @@ app.factory('formRenderingService', ['$window', '$http', function ($window, $htt
         }
     }
 }]);
+
+/**
+ * Service to retrieve the formly template (which is just a static JSON file)
+ * for a given form.
+ */
+app.factory('formTemplateService', ['$http', function ($http) {
```
Comment (ahsu1230) [inline winterfell/webapp/src/js/formRenderingService.js]: There is a service called net.js which handles API endpoints to our server.
You should use that if you want to make HTTP requests

Comment (ahsu1230) [inline winterfell/webapp/src/js/formRenderingService.js]: it adds a wrapper around $http so all you have to provide is a url and body (if needed)

**Comment by ahsu1230** [comment]
other than one comment, looks fine to me

File: winterfell/webapp/src/js/formRenderingService.js (hunk 1 above)
Comment (qwwqwwq) [inline winterfell/webapp/src/js/formRenderingService.js]: Which library is that, I don't see anything called net in the package json? I agree $http is a little kludgy, I think using $resource might be preferred.

Comment (ahsu1230) [inline winterfell/webapp/src/js/formRenderingService.js]: it's not in package.json. It's a custom file on the same level as this file 'webapp/src/js/net.js'.

**Comment by qwwqwwq** [comment]
Decided not to use xhr to get the form schemas, it is a little slow and the schemas are not that big

**Pull request merged**